productivity-suite/piercing-library/src/message-bus/get-bus.ts

```diff
@@ -1,13 +1,17 @@
 import { getFragmentHost } from "../piercing-fragment-host/get-fragment-host";
-import type { MessageBus } from "./message-bus";
+import { GenericMessageBus, type MessageBus } from "./message-bus";
 import { messageBusProp } from "./message-bus-prop";
 
 /**
- * Gets a message bus based on the provided element.
+ * Get a `MessageBus` based on the provided element.
```

Comment:
I think that js/ts docs (unlike commit messages) should be in present simple rather than imperative for example:

![Screenshot 2022-11-28 at 10 51 33](https://user-images.githubusercontent.com/61631103/204260495-92bc98cc-9bd8-4363-9a01-6f9f920275b6.png)

![Screenshot 2022-11-28 at 10 52 12](https://user-images.githubusercontent.com/61631103/204260521-943b35b9-baaf-472a-884f-9c32b6eb3730.png)

Reply:
Yeah - I guess so. I just find it ugly. But I should be consistent with the rest of the world 😄

Reply:
😅👍 

anyways I guess it could be a bit ugly when looking at it in the code, but I find that it does make sense when you see it as IDE suggestions since they are explaining stuff to the developer, imperative would look ugly there, wouldn't it? 🤔